azure-pipelines.yml

```diff
@@ -6,19 +6,19 @@ trigger:
       - master
 
 jobs:
-  - template: azure/job.yml
+  - template: azure/macos/job.yml
```

Comment:
We only seem to be testing one configuration in mac ?

Reply:
Yes, we don't have CI time for more. We should likely use the DEBUG_ZTS build though.

Reply:
Okay, testing DEBUG_ZTS seems acceptable for now ...